lexers/p/python2_test.go

```diff
@@ -0,0 +1,39 @@
+package p_test
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/alecthomas/assert"
+	"github.com/alecthomas/chroma"
+	"github.com/alecthomas/chroma/lexers/p"
+)
+
+func TestCpp_AnalyseText(t *testing.T) {
```

Comment:
small copy&paster leftover
```suggestion
func TestPython2_AnalyseText(t *testing.T) {
```